cs130_front/src/components/views/GroupsPage/GroupItem.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import Text from '../../Text/Text';
+import TextInput from '../../TextInput/TextInput';
+import Button from '../../Button/Button';
+import UserList from '../../UserList/UserList';
+import * as Colors from '../../../constants/Colors';
+import './styles.css';
```

Comment:
+import { withRouter } from 'react-router-dom';

Reply:
Why do I need this?